common/changes/@itwin/core-backend/io-temp-fix-health-stats-test_2025-10-29-16-26.json

```
@@ -0,0 +1,10 @@
+{
+  "changes": [
+    {
+      "packageName": "@itwin/core-backend",
+      "comment": "Temporarily skipping the health check stats test",
```

```suggestion
      "comment": "",
```

We don't need to add test upkeep to the changelog